flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/cache/AbstractDownloadCache.java

```diff
@@ -128,7 +128,11 @@ public Path getOrDownload(final String url, final Path targetDir) throws IOExcep
             Files.createDirectories(scopedDownloadDir);
             log.info("Downloading {}.", url);
             AutoClosableProcess.create(
-                            CommandLineWrapper.wget(url).targetDir(scopedDownloadDir).build())
+                            CommandLineWrapper.wget(url)
+                                    .targetDir(scopedDownloadDir)
+                                    .timeoutSecs(
+                                            downloadGlobalTimeout.getSeconds() / downloadMaxRetries)
```

Comment:
shouldn't this use the `downloadAttemptTimeout`?

Reply:
overlooked it, thx